okhttp/src/test/java/io/grpc/okhttp/OkHttpTlsUpgraderTest.java

```diff
@@ -32,4 +34,44 @@
             || OkHttpTlsUpgrader.TLS_PROTOCOLS.indexOf(Protocol.GRPC_EXP)
                 < OkHttpTlsUpgrader.TLS_PROTOCOLS.indexOf(Protocol.HTTP_2));
   }
+
+  @Test public void upgrade_canonicalizeHost() {
+    // IPv4
+    assertEquals(canonicalizeHost("127.0.0.1"), "127.0.0.1");
```

Comment:
The argument order for `assertEquals` is reversed. These should be in the form: `assertEquals("ip", canonicalizeHost("ip"))`. (this results in better output when a test fails, as error message will label the first argument as the expected value and the second argument as the actual value)